connect/mirror/README.md

```diff
@@ -141,7 +141,38 @@ nearby clusters.
 N.B. that the `--clusters` parameter is not technically required here. MM2 will work fine without it; however, throughput may suffer from "producer lag" between
 data centers, and you may incur unnecessary data transfer costs.
 
-## Shared configuration
+## Configuration
+### General Kafka Connect Config
+All Kafka Connect, Source Connector, Sink Connector configs, as defined in [Kafka official doc] (https://kafka.apache.org/documentation/#connectconfigs), can be 
```

Comment:
The link does not render correctly, remove the space between `] (`

Reply:
done